AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -174,6 +178,42 @@ + (void)setCustomProperties:(MSCustomProperties *)customProperties {
 }
 #endif
 
++ (void)setAuthTokenDelegate:(id<MSAuthTokenDelegate>)authTokenDelegate {
```

Comment:
Where is the set auth token method? just not written yet?

Reply:
This is written now